src/components/Footer/Footer.jsx

```diff
@@ -56,7 +61,7 @@ const Footer = () => {
   const handleReportSubmit = async (e) => {
     e.preventDefault();
 
-    if (!Cookies.get("token")) {
+    if (!Cookies.get("Token")) {
```

Comment:
```suggestion
    if (!Cookies.get("isLoggedIn")) {
```

Please change the `Token` cookie

Reply:
what do you mean by 'change'?